bindings/stronghold-nodejs/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![deny(clippy::all)]
+#![forbid(unsafe_code)]
+
+#[macro_use]
+extern crate napi_derive;
```

Comment:
Nit: I don't think `extern crate` is necessary? We should be able to use its macros via imports.